mmpose/models/backbones/mobilenet_v3.py

```diff
@@ -164,6 +162,8 @@ def forward(self, x):
             x = layer(x)
             if i in self.out_indices:
                 outs.append(x)
+            elif i == (len(self.layers) - 1) and -1 in self.out_indices:
```

Comment:
What about supporting general negative index, like -1, -2, ...?

Reply:
It is good. I will do it.